Clinical-Genomics/cg
# Display family relations

This PR adds relations in cli/get/family

Contact: @emiliaol 

How to test cg get reltions:
0. install on cg stage on hasta (add instructions on how to as needed)
1. us
1. `cg get relations CASE_ID`

Expected outcome:
a table with related samples,
sample-id, mother-sample, father sample is displayed for each sample

How to test cg get family with relations:
0. install on cg stage on hasta (add instructions on how to as needed)
1. us
1. `cg get family CASE_ID`

Expected outcome:
family displayed with 
a table with related samples,
sample-id, mother-sample, father sample is displayed for each sample

How to test cg get family without relations:
0. install on cg stage on hasta (add instructions on how to as needed)
1. us
1. `cg get family --no-relate CASE_ID`

Expected outcome:
family displayed w/o explicit relations


This is a minor **version bump** because it expands the interface in a backwards compatible manner
